activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/RuntimeBundleSwaggerConfig.java

```diff
@@ -52,9 +56,34 @@ public GroupedOpenApi runtimeBundleApi(@Value("${activiti.cloud.swagger.rb-base-
             .addOpenApiCustomizer(openApi ->
                 openApi.addExtension(BaseOpenApiBuilder.SERVICE_URL_PREFIX, swaggerBasePath)
             )
+            .addOpenApiCustomizer(openApiCustomizer())
             .build();
     }
 
+    public OpenApiCustomizer openApiCustomizer() {
+        return openAPI ->
+            openAPI
+                .getPaths()
+                .values()
+                .stream()
+                .flatMap(val -> val.readOperations().stream())
+                .flatMap(operation -> operation.getResponses().entrySet().stream())
+                .filter(entry -> entry.getKey().matches("200"))
```

Comment:
Please use HttpStatus.OK.value() instead of 200

Reply:
Done